src/svelte/src/components/DataDisplays/Header/DisplayHeader.svelte

```diff
@@ -37,28 +36,20 @@ limitations under the License.
   import { UIThemeCSSClass } from '../../../utilities/colorScheme'
   import { createEventDispatcher } from 'svelte'
   import { OffsetSearchType } from '../../Header/fieldsets/SearchReplace'
-
-  type ViewportDivSpread = '24px' | '28px' | '68px'
+  import { byteDivWidthFromRadix } from '../../../utilities/display'
 
   const eventDispatcher = createEventDispatcher()
-  const bitNumText = '01234567'
-  const physicalOffsetSpreads = {
-    16: '24px',
-    10: '28px',
-    8: '28px',
-    2: '68px',
-  }
-
-  let phyiscalOffsetSpread: ViewportDivSpread
+  let bitIndexStr = '01234567'
   let selectionOffsetText: string
-  let offsetLine = []
+  let offsetLine: string[] = []
 
   $: {
     offsetLine = generate_offset_headers(
       $addressRadix,
       $displayRadix,
       $bytesPerRow
     )
+    console.log(offsetLine)
```

Comment:
Nuke this please.

Reply:
I _knew_ that something would get left behind lol